dask/array/tests/test_array_core.py

```diff
@@ -5406,3 +5406,13 @@ def test_load_store_chunk():
     )
     expected = np.array([])
     assert all(actual == expected)
+
+
+def test_interp():
```

Comment:
A few more tests would be great, I'd be happy if you use xarray to call into the functions if that makes things easier fwiw